example/fittemplate.md

```diff
@@ -0,0 +1,157 @@
+    cfg.rotate       = [0 0 0];
+    defaced_template =  ft_defacemesh(cfg, defaced_template);
+
+    cfg              = [];
+    cfg.translate    = [-40 0 -50];
+    cfg.scale        = [200 200 200];
+    cfg.rotate       = [0 0 0];
+    defaced_polhemus =  ft_defacemesh(cfg, polhemus);
+
+Now we have another look at the surfaces
+
+    ft_plot_mesh(defaced_template);  
+    ft_plot_mesh(defaced_polhemus);  
+
+{% include image src="/assets/img/example/fittemplate/defaced_template_surface_polhemus_surface_plot.png" %}
+
+
+## Determining and applying of the transformation
+
+### Method 1: On the basis of spheres fitted to the head shapes
+
+For the first approach we will fit a sphere to the three fiducials and we will fit another sphere to the template head model. On the basis of difference of the center and scaling between the two spheres we can derive a transformation containing information of the translation and the global scaling.
+
+Fit a sphere to the MRI template
+
+    cfg=[];
+    cfg.method='singlesphere';
+    template_sphere = ft_prepare_headmodel(cfg, defaced_template);
+
+Fit a sphere to the polhemus headshape
+
+    cfg=[];
+    cfg.method = 'singlesphere';
+    polhemus_sphere = ft_prepare_headmodel(cfg, defaced_polhemus);
+
+Determine the global scaling and translation
+
+    scale = polhemus_sphere.r/template_sphere.r;
+
+    T2 = [1 0 0 template_sphere.o(1);
+          0 1 0 template_sphere.o(2);
+          0 0 1 template_sphere.o(3);
+          0 0 0 1                ];
+
+    T1 = [1 0 0 -template_sphere.o(1);
+          0 1 0 -template_sphere.o(2);
+          0 0 1 -template_sphere.o(3);
+          0 0 0 1                 ];
+
+    S  = [scale 0 0 0;
+          0 scale 0 0;
+          0 0 scale 0;
+          0 0 0 1 ];
+
+    transformation = T1*S*T2;
+
+Apply the transformation to the template head model
+
+    template_sphere = ft_transform_geometry(transformation, template)
+
+### Method 2: On the basis of the full head surface
+
+This requires an external toolbox, which can be downloaded [here](https://sites.google.com/site/myronenko/research/cpd)
+
+With this method we apply an affine transformation to the template head model to fit the
+whole Polhemus head shape.  This not only applies a translation and rotation, but also a
+scaling in the different directions and some skewing.
+
+We will now use the template scalp and the Polhemus head shape to determine the
+affine transformation.
+
+    cfg              = [];
+    cfg.headshape    = polhemus;
+    cfg.template     = defaced_template;
+    cfg.method       = 'fittemplate';
+    template_surface = ft_prepare_mesh(cfg, template.bnd);
+
+## Preparing the head models
+
+Now that we have have the surfaces of the full template model (not only the scalp) transformed to fit the Polhemus measurement, we can recompute the volume conduction model
+
+    cfg = [];
+    cfg.method = 'openmeeg';
+    headmodel_sphere = ft_prepare_headmodel(cfg, template_sphere.bnd);
```

Comment:
I propose at the end also to visualize both head models (i.e. the outer surface) together with the complete polhemus head shape

Reply:
Added visualisation